src/gc/impl/conservative/gc.d

```diff
@@ -4469,3 +4480,41 @@ unittest
         printf("unexpected pointers %p and %p\n", p.ptr, q.ptr);
     }
 }
+
+// https://issues.dlang.org/show_bug.cgi?id=20256
+version (linux) unittest
+{
+    import core.sys.posix.signal;
+    import core.sys.posix.unistd;
+    import core.thread;
+    import core.memory;
+
+    int sig;
+    sigset_t m, o;
+    sigemptyset(&m);
+    sigaddset(&m, SIGHUP);
+
+    auto x = new int[](10000);
+    for(int i=0;i<10000;i++)
```

Comment:
The style checker expects a space after `for` and a few more for the expressions might be good, too.